pkg/utils/utils.go

```diff
@@ -76,7 +78,13 @@ func GetPfName(pciAddr string) (string, error) {
 		}
 		return "", err
 	} else if len(files) > 0 {
-		return files[0].Name(), nil
+		name, err := sriovnet.GetUplinkRepresentor(pciAddr)
+
+		if err != nil {
+			return files[0].Name(), nil
```

Comment:
why not return the error? as I remember this method sriovnet.GetUplinkRepresentor support both old kernel and new kernel